.env.example

```
@@ -146,6 +146,9 @@ MARGINCE_GMAIL_CLIENT_ID=
 MARGINCE_GMAIL_CLIENT_SECRET=
 MARGINCE_GRAPH_CLIENT_ID=
 MARGINCE_GRAPH_CLIENT_SECRET=
+# Outlook push. The api's token and the worker's URL must carry the same secret.
+MARGINCE_GRAPH_PUSH_TOKEN=
+MARGINCE_GRAPH_NOTIFICATION_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MARGINCE_GRAPH_NOTIFICATION_URL key should go before the MARGINCE_GRAPH_PUSH_TOKEN key

(UnorderedKey)